### PR TITLE
[IMP] website: improves ui interface of items snippet

### PR DESCRIPTION
**Pull request by oussamamessaoudi**
task-2471323

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by JKE-be — CHANGES_REQUESTED** [review]
please, compress picture :/

File: addons/website/views/snippets/s_product_list.xml [hunk 1]
```diff
@@ -2,55 +2,56 @@
 <odoo>
 
 <template name="Items" id="s_product_list">
-    <section class="s_product_list">
+    <t t-set="url" value="url or '#'"/>
+    <section class="s_product_list" >
```
Comment (qsm-odoo) [inline addons/website/views/snippets/s_product_list.xml:6]: Again an useless whitespace added, please remove it

File: addons/website/data/website_data.xml [hunk 2]
```diff
@@ -648,37 +648,37 @@
...
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/services.jpg</field>
         </record>
         <record id="website.s_product_list_default_image_6" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_6.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/multimedia.jpg</field>
```
Comment (qsm-odoo) [inline addons/website/data/website_data.xml:681]: Note: you left the old image in the repo but it does not seems used anywhere... or should we left it there for "compatibility" @JKE-be ? We don't really ensure "theme" / "default" images to be kept anywhere else anyway... so I would simply remove it :grimacing:

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**